### ISSUE TITLE
Post no language-selector show

### ISSUE THREAD

**Issue by nick1122** [thedevdojo/voyager#1063]
- Laravel Version: 5.3.#
- Voyager Version: 0.11.8
- PHP Version:7.0
- Database Driver & Version:

### Description:

Post use Translatable but no language-selector show on

can someone help me ? thanks!

**Comment by akazorg** [comment]
Hi @nick1122, you need to enable translations at `config/voyager.php`. Did you made this?

**Comment by davodavodavo3** [comment]
there is issue
```
ErrorException in Translatable.php line 209:
Argument 2 passed to TCG\Voyager\Models\Post::setAttributeTranslations() must be of the type array, null given
```

**Comment by akazorg** [comment]
hi @davodavodavo3, please open another issue and show us the full error description, also tell us the context how it happen.
~~Posts translations are working.~~

[edit]
Guys, I'm sorry, you're absolutely right.
I was checking and that's missing. I'll create a PR for that ASAP!!

**Comment by eddielee394** [comment]
Can also confirm the issue.  I've literally spent the last week trying to diagnose it and ended up having to build out my own localization method because I had a tight deadline on the project.  Bummer too, because I was really hoping to use the Voyager native localization.  Next project I guess.  

`Argument 2 passed to TCG\Voyager\Models\Post::setAttributeTranslations() must be of the type array, null given, called in E:\Backup\wp-dev\htdocs\xxx.dev\vendor\tcg\voyager\src\Http\Controllers\Controller.php on line 329 and defined`

**Comment by davodavodavo3** [comment]
@eddielee394 there is a fix for that issue in realese 0.11.9

**Comment by akazorg** [comment]
Fixed on #1148

**Comment by DrudgeRajen** [comment]
#3601
